frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -178,6 +178,27 @@ describe("scenarios > admin > settings", () => {
     cy.contains(/^February 11, 2019, 9:40 PM$/);
   });
 
+  it("should search for and select a new timezone", () => {
+    cy.server();
+    cy.route("PUT", "**/report-timezone").as("reportTimezone");
+
+    cy.visit("/admin/settings/localization");
+    cy.contains("Report Timezone")
+      .closest("li")
+      .find(".AdminSelect")
+      .click();
+
+    cy.focused().type("Centr");
```

Comment:
```suggestion
    cy.findByPlaceholderText("Find...").type("Centr");
```
There are really helpful addons for Cypress ([`@testing-library/cypress`](https://github.com/testing-library/cypress-testing-library)) such as this one.

`cy.focused` was ok for this and will probably keep working, but what if that input doesn't receive focus in the future (for whatever reason)?

Another alternative would be:
```javascript
popover().within(() => {
  cy.get("input").type("Centr");
});
```

But you can see how much more elegant `cy.findByPlaceholderText()` is :)